src/main/kotlin/no/nav/klage/oppgave/api/OppgaveController.kt

```diff
@@ -42,4 +42,26 @@ class OppgaveController(val oppgaveService: OppgaveService) {
             )
         }
     }
+
+    @GetMapping("/oppgaver/{id}")
+    fun getOppgave(@PathVariable oppgaveId: Int): OppgaveView {
```

Comment:
Funker dette selv om {id} og oppgaveId ikke matcher? Er vant til at de skal være like for at det skal funke. Men kanskje funker om det bare er én parameter?

Reply:
Godt poeng - jeg burde skrevet @PathVariable("id")